lib/api/controllers/serverController.js

```diff
@@ -54,6 +55,27 @@ class ServerController extends NativeController {
       json: null,
       yaml: null,
     };
+    this._info = {
+      elasticsearch: {},
+      redis: {}
+    };
+  }
+
+  /**
+   * Init ServerController
+   * Used to perform asynchronous initialization safely: the funnel will wait
+   * for all controllers to be initialized before accepting requests.
+   * 
+   * @returns {Promise}
+  */
+  async init () {
+    const [storageInfo, cacheInfo] = await Promise.all([
+      global.kuzzle.ask('core:storage:public:info:get'),
+      global.kuzzle.ask('core:cache:public:info:get'),
+    ]);
+    this._info.elasticsearch = storageInfo;
+    this._info.redis = cacheInfo;
+    return Bluebird.resolve();
```

Comment:
You don't need to return another promise, since the method is `async` it will automatically returns a promise